### PR TITLE
Feat: V0.91 Add drug product information

### PR DESCRIPTION
**Pull request by piehld**
Feat: V0.91 Add drug product information (approved, country, dates of marketing, etc.).

Jira ticket: https://rcsbpdb.atlassian.net/browse/RO-3975

### CODE REVIEW

**Comment by piehld** [comment]
Hi @brindakv, I tested this using your schemas at https://github.com/rcsb/py-rcsb_exdb_assets/pull/130, and loading worked great.

**Comment by piehld** [comment]
(Also, I used your py-rcsb_db code at https://github.com/rcsb/py-rcsb_db/pull/67).

File: rcsb/utils/chemref/DrugBankReader.py [hunk 1]
```diff
@@ -173,6 +181,35 @@ def __getTargetInfo(self, category, target):
...
+                    elif v == "false":
+                        dpD["approved"] = "N"
+                else:
+                    dpD[k] = v
+            dpL.append(dpD)
+        #
+        drugProductL = self.__removeDuplicateDicts(dpL)
+        return drugProductL
+
+    def __removeDuplicateDicts(self, dictList, key=lambda d: tuple(sorted(d.items()))):
```
Comment (brindakv) [inline rcsb/utils/chemref/DrugBankReader.py:217]: @piehld Similar to `drug_products`, `drug_groups` also have `uniqueItems` set to `true` in the schema. Perhaps good to address that as well.

**Review by brindakv — CHANGES_REQUESTED** [review]
@piehld Thanks. Looks good overall. Please see my comment below.

**Comment by brindakv** [comment]
@piehld I also looked at the documents loaded into the test mongo instance, and they look good.

**Review by brindakv — APPROVED** [review]
Thanks @piehld. Looks good.

File: rcsb/utils/chemref/DrugBankReader.py (hunk 1 above)
Comment (piehld) [reply rcsb/utils/chemref/DrugBankReader.py:217]: Thanks @brindakv. I don't think there's a chance `drug_groups` would have duplicate items, but I've added an additional bit of code to ensure that as well.

**Pull request merged**